### DISCUSSION TITLE
No configured loader

### DISCUSSION THREAD

**Discussion by SciManJD** [bcakmakoglu/vue-flow#851] [Q&A] [answered]
I'm trying to use vue-flow component in my application which is using electron-builder. I always get error:

 error  in ./node_modules/@vue-flow/core/dist/vue-flow-core.mjs

Module parse failed: Unexpected token (78:82)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     if (!Er(g))
|       return n((x) => x === g, v);
>     const { flush: w = "sync", deep: f = !1, timeout: d, throwOnTimeout: _ } = v ?? {};
|     let y = null;
|     const A = [new Promise((x) => {

I can't solve this problem. Give me an advice, please...

**Comment by bcakmakoglu** [comment]
Could you try updating to `1.19.0` and see if that helps?

**Reply by SciManJD** [reply]
This is exactly the version that I'm using. The problem occurs. What can I do?

**Reply by bcakmakoglu** [reply]
When you posted this that version wasn't even released 🤔
Have you tried removing your node_modules & package-lock and reinstalling from scratch?

**Reply by SciManJD** [reply]
Ok. I tried to give a short answer, but it wasn't clear enoug... A week ago I installed the component and I had this problem. I didn't check the version. Then I posted my question here, uninstalled the component and tried to find another solution. Unfortunately there is no big choice for Vue 3. When I saw the answer today, I installed again the component (I checked the version this time and it was exactly 1.19.0 so I though that it is the same as before). Nevertheless, the result is the same. And I'm sure that the version now is 1.19.0. I really need such component, so please, help me...

**Reply by bcakmakoglu** [reply]
Can you create a reproduction for this?

A repo or a sandbox would help debug the issue and provide you some proper help.

**Reply by SciManJD** [reply]
I will see what can I do. When I have a result, I will write again.

**Reply by bcakmakoglu** [reply]
Maybe this thread can help you out:
https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/67

**Reply by SciManJD** [reply]
I tried to create a simple Vue 3 project that uses VueFlow. I had no problems to compile it. Then I installed all components that I used in my original project. I also tried to equalize (as much as possible) the configuration files. I cannot reproduce the problem, I understand that I didn't help you to locate the problem, but I don't know what can I do. My project is very complex and contains a huge code. Do you have an idea how can I provide you more information for finding the problem? Maybe some logs somewhere?

**Reply by SciManJD** [reply]
I found something else. The problem comes from the nullish coalescing operator ??. If I correct one line, I get an error on another line with this operator. As far as I understand, this is relatively new operator in Typescript (sorry, but I'm not familiar with typescript at all). So it seems that this is some kind of configuration problem... Now can you guess what can be the reason? Meantime I continue searching information from Internet what can cause the problem...

**Reply by bcakmakoglu** [reply]
It's not really typescript related as the file your project is using is an `mjs` file (a module js file); It seems more like your app just doesn't handle this file-type and you'd have to either transpile it yourself. Have you checked the link I posted previously? Maybe one of the proposed solutions in there works? ^^

**Reply by SciManJD** [reply]
I tried what is described in the link, but there is no difference. I continue trying and reading in Internet how to fix it. If you have another idea, please, write. At least now it is clear that this is something specific that happens in my project.

**Reply by SciManJD** [reply, marked as answer]
I fixed the problem. I got the solution from Internet. In my vue.config.js file I added the line:

transpileDependencies: ['@vue-flow/core']

The error disappeared and it seems that the component is working.

**Reply by bcakmakoglu** [reply]
Nice :)

**Reply by bcakmakoglu** [reply]
If you can link the source for this, that would possibly help others facing the same problem

**Reply by SciManJD** [reply]
Sorry that I didn't publish this link immediately, but now I'm not sure which one is it. I read a lot of topics and my browser's history doesn't help too much. Nevertheless, I added only this line exactly how I wrote it above. There is no other change.
